### ISSUE TITLE
Use "chromosome" instead of "contig" throughout the user interface

### ISSUE THREAD

**Issue by armish** [hammerlab/cycledash#701]
The term `contig` is really not well-known to biologists and therefore might be confusing for many of the Cycledash users. I suggest we use `chromosome` instead of `contig` throughout the user interface and also in the CQL before it gets being used by researchers.

This is also relevant to [pileup.js](https://github.com/hammerlab/pileup.js) terminology.

What do people think?

Here are some relevant tools and use cases for better comparison:

- This is the default [BED](https://genome.ucsc.edu/FAQ/FAQformat.html#format1) file format from [Genome Browser](https://genome.ucsc.edu):
![screen shot 2015-06-08 at 6 51 20 pm](https://cloud.githubusercontent.com/assets/7809/8047127/7502665a-0e0f-11e5-9403-6afa1a45c9c2.png)

- This is a screenshot from [the browser](https://genome.ucsc.edu) itself:
![screen shot 2015-06-08 at 6 50 05 pm](https://cloud.githubusercontent.com/assets/7809/8047163/e6a20c5c-0e0f-11e5-8e93-f19a1244314c.png)

- These are from IGV:
![screen shot 2015-06-08 at 6 50 59 pm](https://cloud.githubusercontent.com/assets/7809/8047128/7506916c-0e0f-11e5-8f95-65199d4adff2.png)
![screen shot 2015-06-08 at 6 51 08 pm](https://cloud.githubusercontent.com/assets/7809/8047129/75085bfa-0e0f-11e5-9a69-0c8a904e7909.png)

- And this is Cycledash:
![screen shot 2015-06-08 at 6 41 36 pm](https://cloud.githubusercontent.com/assets/7809/8047173/fe50f606-0e0f-11e5-8132-afcfe2068d77.png)

**Comment by ihodes** [comment]
Something worth some further user-validation @jaclynperrone 

I think the reason I changed it to/settled on contig was because we're not just dealing with chromosomes; we're working with many unmapped contigs, bacterial contigs, mitochondrial DNA, etc.

**Comment by danvk** [comment]
My 2¢: "contig" is more general and more correct, but 99% of the time people are going to be dealing with true chromosomes, particularly when they're working with human DNA. I don't think we'd lose much by using the term "chromosome". Even mitochondrial DNA is referred to as a "bacterial chromosome" and is often marked as `chrM` in VCFs and BAMs.

**Comment by armish** [comment]
I didn't even know that Cycledash could handle assemblies other than human. In that sense, I agree that `contig` is the right term to use, but it is one of those things people get quite confused about. Might make sense to wait until @jaclynperrone validates my assumption with the upcoming interviews.

**Comment by ihodes** [comment]
@armish yeah I think we're confining ourselves to human DNA for the time being, but we often see contigs other than the 23 + M chromosomes in human DNA. Can't pull up Cycledash (DNS!) to get an example now, though.
